### PR TITLE
feat: Adds line chart types

### PR DESCRIPTION
**Pull request by Golodhros**
## Description

Adds types for the line chart (https://github.com/Golodhros/britecharts/blob/master/src/charts/line.js#L47)
Orders index alphabetically

## Motivation and Context

Completing types for Britecharts

## How Has This Been Tested?

Manually on the sandbox

## Screenshots (if appropriate):
Britecharts_test_for_typescript_

## Types of changes

- [ ] Refactor (changes the way we code something without changing its functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] Latest master code has been merged into this branch
- [ ] No commented out code (if required, place // TODO above with explanation)
- [ ] No linting issues
- [ ] Build is successful
- [ ] Updated the documentation
- [ ] Added tests to cover changes
- [ ] All new and existing tests passed

### CODE REVIEW

File: src/typings/charts/line-chart.d.ts [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  grid(gridType?: GridTypes): LineChartModule;
+  lineCurve(curveType?: string): LineChartModule;
+  lineGradient(gradient?: [string, string]): LineChartModule;
+  lines(customLines: CustomLine[]): LineChartModule;
+  shouldShowAllDataPoints(showAllDataPoints?: boolean): LineChartModule;
+  tooltipThreshold(threshold?: number): LineChartModule;
+  xAxisCustomFormat(format?: string): LineChartModule;
+  xAxisFormat(format?: string): LineChartModule;
+  xAxisLabel(label: string): LineChartModule;
+  xAxisScale(scale?: 'linear' | 'logarithmic'): LineChartModule;
```
Comment (DalerAsrorov) [inline src/typings/charts/line-chart.d.ts:52]: Are these the only scales available?

File: src/typings/charts/line-chart.d.ts [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  xAxisValueType(valueType?: 'date' | 'numeric'): LineChartModule;
+  xTicks(ticks: number): LineChartModule;
+  yAxisLabel(label: string): LineChartModule;
+  yAxisLabelPadding(padding: number): LineChartModule;
+  yTicks(ticks: number): LineChartModule;
+}
+// exportChart
```
Comment (DalerAsrorov) [inline src/typings/charts/line-chart.d.ts:59]: not sure if this comment was intended?

File: src/typings/charts/line-chart.d.ts (hunk 1 above)
Comment (Golodhros) [reply src/typings/charts/line-chart.d.ts:52]: Yep!

File: src/typings/charts/line-chart.d.ts (hunk 2 above)
Comment (Golodhros) [reply src/typings/charts/line-chart.d.ts:59]: Yes, a reminder for later

**Review by DalerAsrorov — APPROVED** [review]

**Pull request merged**